cli-copy.test.js

```diff
@@ -0,0 +1,36 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const { HELP_OPTIONS } = require("./_modules/constants");
+const testUtils        = require("./tests/test-utils");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("cli-copy", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region -h, --help
+    // -----------------------------------------------------------------------------------------
+
+    describe(HELP_OPTIONS, () => {
+        test.each`
+            option
+            ${"-h"}
```

Comment:
Wondering if HELP_OPTIONS or this setup can be reworked so `-h` and `--help` don't need to be typed out here being they are technically a comma-delimited string in `HELP_OPTIONS` itself. Perhaps HELP_OPTIONS is an array?

Reply:
Updated to be an array 👍